src/platform/FreeBSD.cc

```diff
@@ -187,8 +212,23 @@ void FreeBSD::Execute(const CommandLine& c) const
 			libs += ":";
 		}
 	}
+
 	setenv("LD_LIBRARY_PATH_FDS", libs.c_str(), 1);
 
+    int shmfd = po_pack(map);
```

Comment:
This is looking much better now. Please ensure that you use tabs instead of four spaces, however.